### PR TITLE
feat(Task-3): As a developer, I want to manage the screen change easily

### PR DESCRIPTION
**Pull request by jLagosCarrera**
Changes made on #1 will be on this branch when merging into development branch

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+<head>
+    <meta charset="utf-8">
+    <title>Hello AngularJS!</title>
+</head>
+
+<body ng-app="app" ng-strict-di ng-cloak>
+    <a ui-sref="hello" ui-sref-active="active">Hello</a>
+    <a ui-sref="world" ui-sref-active="active">World</a>
+    <br />
```
Comment (eloydotco) [inline src/index.html]:
```suggestion
    <br>
```

File: src/app/app.module.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import helloComponent from './hello/hello';
+import worldComponent from './world/world';
+
+const app = angular.module('app', [uiRouter]);
+
+app.component('hello', helloComponent);
+app.component('world', worldComponent);
+
+app.config(['$stateProvider', function (provider) {
+    var helloState = {
```
Comment (eloydotco) [inline src/app/app.module.js]: User const/let instead of var

File: src/app/app.module.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+        name: 'hello',
+        url: '/hello',
+        component: 'hello'
+    }
+
+    var constState = {
```
Comment (eloydotco) [inline src/app/app.module.js]: same as above

File: src/app/app.module.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+import uiRouter from '@uirouter/angularjs';
+import helloComponent from './hello/hello';
+import worldComponent from './world/world';
+
+const app = angular.module('app', [uiRouter]);
+
+app.component('hello', helloComponent);
+app.component('world', worldComponent);
+
+app.config(['$stateProvider', function (provider) {
```
Comment (eloydotco) [inline src/app/app.module.js]: Just fyi, this routing/states configuration is on its own file and imported on module.js
maybe for the future is a good refactoring 👍

**Review by eloydotco — APPROVED** [review]

**Comment by eloydotco** [comment]
Also, solve the conflicts.

**Comment by jLagosCarrera** [comment]
Requested changes made 🤘

File: src/app/component/component.js [hunk 5]
```diff
@@ -0,0 +1,8 @@
+import template from './component.html';
+import './component.scss';
+
+const AppComponent = {
+    template
+};
+
+export default AppComponent;
```
Comment (gbtec-ext-jorgevillar) [inline src/app/component/component.js]: I think that our way of creating component is shorter and more readable. You may try it
image

**Review by gbtec-ext-jorgevillar — APPROVED** [review]

**Pull request merged**